src/vue-parallax-js.js

```diff
@@ -50,6 +50,7 @@ ParallaxJS.prototype = {
   update() {
     this.items.forEach(function(item) {
       let t = item.el;
+      let n;
```

Comment:
should be `const`, because  `n` is never reassigned (fixed this in #16 )

Reply:
upvote to merge this